<H2>Description</H2>
<P>
This track shows alignments of
<em>Ricinus communis</em> (ricCom1, Jul. 2011)
to <em>Arabidopsis thaliana</em> (araTha1, May 2011)
</P>

<H2>Methods</H2>
<P>
The alignment was performed with
<A HREF="http://www.kentinformatics.com/" TARGET=_blank>
blat</A> with parameters:
<PRE>
-tileSize=11 -minScore=100 -minIdentity=98 -fastMap -ooc=araTha1.11.ooc
</PRE>

Each genome was partitioned into 5,000 base sequences with an overlap
of 500 bases for the target sequence, and each part
from target (araTha1) and query (ricCom1) were blatted to each other.
The araTha1.11.ooc file was created:
<PRE>
blat araTha1.2bit /dev/null /dev/null -tileSize=11 \
      -makeOoc=araTha1.11.ooc -repMatch=100
</PRE>

The resulting alignments were 'chained' together for each araTha1 chromosome
running 'axtChain' selecting chains with at least a minimum score of 1000 and
linearGap 'medium' scoring matrix.
Chain-Net results are created from the chained alignments with 'chainNet'
and then 'liftOver' chain subsets extracted from those chain-net results
with 'netChainSubset:
<PRE>
for C in chr1 chr2 chr3 chr3 chr5 chrCp chrMt
do
cat blatResults.${C}.*.psl \
  | axtChain -verbose=0 -linearGap=medium -psl stdin \
      araTha1.2bit ricCom1.2bit ${C}.chain
chainNet ${C}.chain \
    araTha1.chrom.sizes ricCom1.chrom.sizes \
    ${C}.net /dev/null
netChainSubset ${C}.net ${C}.chain stdout \
  | chainStitchId stdin stdout
done &gt; araTha1.ricCom1.over.chain
</PRE>

The 'liftOver' chains were converted to a bigBed file via 'chainToPsl',
'pslToBed' and 'bedToBigBed':
<PRE>
chainToPsl -tMasked araTha1.ricCom1.over.chain \
  araTha1.chrom.sizes ricCom1.chrom.sizes araTha1.2bit ricCom1.2bit stdout \
    | pslToBed stdin stdout | sort -k1,1 -k2,2n &gt; araTha1ToRicCom1.bed

bedToBigBed araTha1ToRicCom1.bed araTha1.chrom.sizes araTha1ToRicCom1.bb
</PRE>
</P>

<h2>References</h2>

<P>
Chiaromonte F, Yap VB, Miller W.
<A HREF="http://psb.stanford.edu/psb-online/proceedings/psb02/chiaromonte.pdf"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<em>Pac Symp Biocomput</em>. 2002:115-26.
PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/11928468" target="_blank">11928468</a>
</p>

<P>
Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D.
<A HREF="http://www.pnas.org/content/100/20/11484.abstract"
TARGET=_blank>Evolution's cauldron:
duplication, deletion, and rearrangement in the mouse and human genomes</A>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Sep 30;100(20):11484-9.
PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/14500911" target="_blank">14500911</a>; PMC: <a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC208784/" target="_blank">PMC208784</a>
</p>

<P>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC,
Haussler D, Miller W.
<A HREF="http://genome.cshlp.org/content/13/1/103.abstract"
TARGET=_blank>Human-mouse alignments with BLASTZ</A>.
<em>Genome Res</em>. 2003 Jan;13(1):103-7.
PMID: <a href="http://www.ncbi.nlm.nih.gov/pubmed/12529312" target="_blank">12529312</a>; PMC: <a
href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC430961/" target="_blank">PMC430961</a>
</p>

